controllers/storagecluster/exporter.go

```diff
@@ -18,18 +19,20 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/utils/ptr"
+	ctrl "sigs.k8s.io/controller-runtime"
 	controllerutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 const (
-	metricsExporterName     = "ocs-metrics-exporter"
-	prometheusRoleName      = "ocs-metrics-svc"
-	metricsExporterRoleName = metricsExporterName
-	portMetricsMain         = "https-main"
-	portMetricsSelf         = "https-self"
-	metricsPath             = "/metrics"
-	rbdMirrorMetricsPath    = "/metrics/rbd-mirror"
-	scrapeInterval          = "1m"
+	metricsExporterName              = "ocs-metrics-exporter"
+	prometheusRoleName               = "ocs-metrics-svc"
+	metricsExporterRoleName          = metricsExporterName
+	portMetricsMain                  = "https-main"
+	portMetricsSelf                  = "https-self"
+	metricsPath                      = "/metrics"
+	rbdMirrorMetricsPath             = "/metrics/rbd-mirror"
+	scrapeInterval                   = "1m"
+	ocsMetricsExporterCephClientName = "ocs-metrics-exporter-ceph-auth"
```

Comment:
in rbd-mirror.go we have hardcoded `ocs-metrics-exporter-ceph-auth` we should move this to util and update the rbd-mirror.go in metrics folder as well